redux/redux.d.ts

```diff
@@ -4,46 +4,77 @@
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
 declare module Redux {
+    // FSA-compliant action (from redux-actions.d.ts)
+    // See: https://github.com/acdlite/flux-standard-action
+    type Action = {
+        type: string
+        payload?: any
+        error?: boolean
+        meta?: any
+    };
```

Comment:
As @wbuchwalter mentioned, FSA should not be enforced here, only type is required. The examples in redux itself would not work with that for example.